### DISCUSSION TITLE
Soporte para guias

### DISCUSSION THREAD

**Discussion by AllenBP** [ChrisK106/sunat-xml-to-pdf#9] [Ideas]
Seria bueno tener un soporte para el XML de guías electrónicas, y poder dar un formato al reporte (como tamaño de fuente, entre otros)

**Comment by ChrisK106** [comment]
> Seria bueno tener un soporte para el XML de guías electrónicas, y poder dar un formato al reporte (como tamaño de fuente, entre otros)

Hola, gracias por tu comentario, lo tomaré en cuenta para una siguiente actualización. :)
